### DISCUSSION TITLE
How to block Minio to connect dl.min.io

### DISCUSSION THREAD

**Discussion by szabokacsa** [minio/minio#15418] [Q&A] [answered]
Hi,
Can you tell me how to disable (turn off) Minio to connect to the dl.min.io web site for security reasons?
What is the purpose that minio connecting to this web site?
Is it mandatory in Minio point of view to connect to this web site?

Minio is running in kubernetes container deployed with kubernetes Deployment like:
![image](https://user-images.githubusercontent.com/3647735/181478527-fcd88ee7-65f1-4682-a11f-cfe7cec9e65f.png)

Is there a way to modify the Deployment file to configure minio do not connect to this web site?

**Comment by klauspost** [comment, marked as answer]
Set `MINIO_UPDATE=off` env var.

**Reply by szabokacsa** [reply]
Thanks!
